### DISCUSSION TITLE
使用犯人的 DNA 畫出沒看過的犯人的畫像

### DISCUSSION THREAD

**Discussion by danwan1027** [1101-GS4719-C/Python-Community-GS4719-C#766] [科技新知]
加拿大的 Edmonton Police Service  首次使用嫌犯的 DNA 來嘗試畫出嫌犯的臉，沒想到 AI 已經進步到這個地步了，但由 AI 畫出的畫像的可靠性仍有待商確。
附上原文：
https://www.vice.com/en/article/pkgma8/police-are-using-dna-to-generate-3d-images-of-suspects-theyve-never-seen?utm_source=reddit.com&utm_source=reddit.com

**Comment by Chiway1229** [comment]
這項科技對於犯罪偵破有一大進展，不過因黑天鵝事件而導致人被關冤獄的機會好像也提升了
可能要等這項技術更為成熟才能實際拿來應用

**Comment by a45320614** [comment]
我覺得文章內第二張比對照片其實還蠻像的XD
不過這項科技還處於起步階段，AI可能還要經過許多訓練才能提供警方使用

**Comment by williamyen0623** [comment]
雖然這項科技的進步能對偵查犯罪是一大幫助，但也有可能因為誤差而抓錯人

**Comment by Rodrigo-dev-bit** [comment]
之前有一個很有名的嫌犯就是用親戚的DNA找到的. https://m.youtube.com/watch?v=KT18KJouHWg&feature=youtu.be

**Comment by manipulateed** [comment]
想不到還可以運用在這個方面，但高正確性可是未來的研究目標。

**Comment by ching020721** [comment]
大部分的相似性感覺沒有很高(主觀)，但代表著這項科技還有很大的進步空間

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
相信這項技術對未來偵辦犯人會有很大的幫助

**Comment by evanchou1204** [comment]
感覺誤差還是偏大，而且像文中說的，可能會加劇種族歧視和對立，應該要謹慎使用。

**Comment by Pietra1226** [comment]
誤差確實偏大，感覺跟測謊器一樣只能參考不能當作判案依據

**Comment by ghost** [comment]
這項技術確實能成為警方的辦案的參考，但如果遇到整容過的人，就可能就會被誤導。所以還是要謹慎使用。

**Comment by Yunhans** [comment]
之後會不會連頭髮都不能隨便掉在路上了

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
這項技術的發明應該能大大的減少冤獄的發生 讓犯人們付起自己該付的責任

**Comment by CPCpyt** [comment]
我覺得這個概念非常棒，或許在日後能為偵辦取得突破的進展。

**Comment by StevenWanglolz** [comment]
這項技術還需要再精進一些才能實際應用吧，連人類都會誤判導致無辜的人被監禁

**Comment by peggy0329** [comment]
我覺得將AI技術用在這種攸關人們自由權的地方不太合適
就算未來技術再發達，畫出正確犯人的正確率高達99%
但100個人中有一個人被冤望我認為還是不太好的，畢竟這攸關人權問題

**Comment by Evan0979** [comment]
感覺這個技術非常好,但是隨著整形技術越來越好......

**Comment by harrylee1971** [comment]
但基因的正確度不高，只能夠僅供參考，不太能成為破案非常重要的證據

**Comment by zephyrxin01** [comment]
這讓我想到一部電影叫做關鍵報告，在一個人工智慧已經厲害到可以預判一個人是否會犯罪的時代，我們全然的相信他到底會帶來什麼後果。但的確的技術可以降低犯罪率並增進社會福祉，那麼我們該對系統提出質疑嗎。

**Comment by sumairu0727** [comment]
關於AI在司法上應用，曾經有法院利用AI判斷該犯人出獄後的再犯率，正確率挺高的，甚至比法官評估的更精準。
即使如此，我們應該思考即使AI的判斷正確率高於法官，我們是否要將「人類」的自由交給無生命的軟體來判斷。

**Comment by Brownnnn78** [comment]
利用DNA找到跟他長得像的人例如家人，搜集長相可能相似的人，或是如果有人看過這個人，也可以人為判斷這個AI做出來的臉是不是和本人像的，或許這樣不同階段的驗證加上AI的學習能力或許有一天能夠做得更像，但用此方法可能可以幫助找到嫌疑人物但用此作為審判依據顯然是不合理的。
